cmd/lncli/commands.go

```diff
@@ -1982,10 +1994,10 @@ var updateChannelPolicyCommand = cli.Command{
 	Action: actionDecorator(updateChannelPolicy),
 }
 
-func parseChanPoint(s string) (*lnrpc.ChannelPoint, error) {
+func parseChannelPointStr(s string) (*lnrpc.ChannelPoint, error) {
```

Comment:
Is there a reason for the rename? Since the first and only argument is a string, isn't it kind of redundant to also have `Str` in the name?

Reply:
+1, I think the existing name is ok!

Reply:
done